CMakeLists.txt

```diff
@@ -559,10 +559,7 @@ else()
   message(STATUS "Not using atomic builtins - user disabled")
 endif()
 
-find_library(VORBISFILE_LIBRARY vorbisfile)
-#check_include_file(libintl.h LIBINTL_HEADER)
-find_library(LIBINTL_LIBRARY intl HINTS /usr/local/lib)
-find_package(Gettext)
+
```

Comment:
This line is unrelated, but `VORBISFILE_LIBRARY` isn't used anywhere, figured I'd remove it while I was here